chipsec/chipset.py

```diff
@@ -384,6 +392,9 @@ def init_xml_configuration(self):
                                 CHIPSET_FAMILY_XEON.append(_cfg.attrib['platform'].upper())
                             if _info.attrib['family'].lower() == "quark":
                                 CHIPSET_FAMILY_QUARK.append(_cfg.attrib['platform'].upper())
+                        if 'detection_value' in _info.attrib:
+                            for dv in list(_info.attrib['detection_value'].split(', ')):
```

Comment:
It might be better to strip spaces just in case.  How about this...  `for dv in [value.strip() for value in _info.attrib['detection_value'].split(',')]:`

Reply:
updated to  `for dv.strip() in list(_info.attrib['detection_value'].split(','))`.  Believe that they are equivalent.